clean_project/construct_dict.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+"""
+Construct a JSON file used to indexing all words in a config_file.yaml
+"""
+import argparse
+import codecs
+import json
+import os
+import yaml
+
+from collections import defaultdict
+from clean_file import sanitize
+
+
+def construct_dictionary(yaml_path, path_json_file):
+    """
+    Construct the dictionary, given a configuration files
+    :param yaml_path: Yaml's path that contains a list of files that will be cleaned
```

Comment:
similarly as below, this should be `cfg_path` or `config_path`. you may migrate to xml config files, or json, or whatever. what doesn't change is the fact that it's a config file. Also, it may not contain just a list of files, but a lot of things, so I'd say that it contains the configuration for the indexer